### ISSUE TITLE
Can you add Google cloud texttospeech ?

### ISSUE THREAD

**Issue by LeprovostNoam** [Aymkdn/assistant-notifier#9]
https://cloud.google.com/text-to-speech/docs/basics?hl=fr

it's return base64 file

**Comment by Aymkdn** [comment]
Hi,

This project is not my priority right now. I'll have a look at your request when I can! I quickly checked and I think it should be possible. But I need to investigate more.

Thanks

**Comment by Aymkdn** [comment]
It's now published (v2.1.7)

I'd assume you already have a Google Cloud account correctly configured, then you have to [create an API Key for Google Text-to-Speech](https://console.cloud.google.com/flows/enableapi?apiid=texttospeech.googleapis.com).

Next, find a voice from https://cloud.google.com/text-to-speech/docs/voices (e.g. *fr-FR-Wavenet-B*) and in your `configuration,json` you have to use the `voice` option:
```javascript
"plugins": {
  "notifier": {
    "host": "192.168.0.13",
    "voice":{
      "source":"google-cloud/text-to-speech@fr-FR-Wavenet-B",
      "key":"API key"
    }
  }
}
```
